app/templates/service_status_update_audits.html

```diff
@@ -0,0 +1,83 @@
+{% import "toolkit/summary-table.html" as summary %}
+{% from 'macros/page_heading.html' import page_heading %}
+
+{% extends "_base_page.html" %}
+{% block page_title %}
+  Digital Marketplace admin
+{% endblock %}
+
+{% set csrf = csrf_token() %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": url_for('.index'),
+        "label": "Admin home"
+      },
+      {
+        "label": "Audits",
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  {% with
+    heading="Service status changes",
+    smaller=True
+  %}
+    {% include "toolkit/page-heading.html" %}
+  {% endwith %}
+  {{summary.heading(day_as_datetime|dateformat)}}
+  {% call(item) summary.list_table(
+    audit_events,
+    caption=day_as_datetime|dateformat,
+    empty_message="No changes",
+    field_headings=[
+      'Supplier', 'User', 'Change', 'Service', 'At'
+    ],
+    field_headings_visible=True
+  ) %}
+    {% call summary.row() %}
+      {% call summary.field(first=True) %}
+        {{item.data.supplierName}}
+      {% endcall %}
+      {{ summary.text(item.user) }}
+      {% call summary.field() %}
+        {{ 'Live' if item.data.new_status == 'published' else 'Removed' }}
+      {% endcall %}
+      {% call summary.field() %}
+        <a href="{{ '/services/g-cloud/{}'.format(item.data.serviceId) }}">{{ item.data.serviceId }}</a>
+      {% endcall %}
+      {{ summary.text(item.createdAt|timeformat) }}
+    {% endcall %}
+  {% endcall %}
+
+  {%
+    with
+    previous_page = {
+      "url": url_for(".service_status_update_audits", day=day, page=page-1),
+      "title": "Page {}".format(page - 1),
+      "label": "of {}".format(day_as_datetime|dateformat)
+    } if previous_page else {
+      "url": url_for(".service_status_update_audits", day=next_day.strftime('%Y-%m-%d')),
+      "title": "Next day",
```

Comment:
I would probably have put the arrows the other way round here: `Next day` to the right and `Previous day` to the left.